docs/howto/outside_controller_use.md

```diff
@@ -20,6 +20,7 @@ serializable_resource = ActiveModelSerializers::SerializableResource.new(post, o
 model_json = serializable_resource.as_json
 ```
 The object that is passed to `ActiveModelSerializers::SerializableResource.new` can be a single resource or a collection.
+The additional options are the same options that are passed [through controllers](https://github.com/rails-api/active_model_serializers/blob/master/docs/general/rendering.md#explicit-serializer).
```

Comment:
@bf4 edited to use relative link in last commit. Yeah I think this is clear and should hopefully help other people who may have had same confusion as me. Thanks